UNRELEASED.md

```diff
@@ -14,6 +14,7 @@
 - Fixed focus state color on monochrome `Buttons` ([#2684](https://github.com/Shopify/polaris-react/pull/2684))
 - Fixed container's width on `Modal` ([#2692](https://github.com/Shopify/polaris-react/pull/2692))
 - Fixed the position property for the backdrop on `Select` from being overwritten by the focus ring ([#2748](https://github.com/Shopify/polaris-react/pull/2748))
+- Fixed the `Popover` reference position calculation ([#2752](https://github.com/Shopify/polaris-react/issues/2752)) ([#1415](https://github.com/Shopify/polaris-react/issues/1415))
```

Comment:
```suggestion
```

Reply:
This should probably go into the "Enhancements" section. Also, the link simply points to the PR as opposed to the issue.

- Added a `preferInputActivator` prop to `Popover` to allow better positioning of the overlay ([#2754](https://github.com/Shopify/polaris-react/pull/2754))

Reply:
It's technically (also) a bugfix for https://github.com/Shopify/polaris-react/issues/2752, but I have no issues moving it in the enhancements section. Thanks for the hint about linking to the issue. I wasn't aware that it should point to that.